src/workers/newNotificationV2Mail.ts

```diff
@@ -695,6 +697,7 @@ const worker: Worker = {
         },
         'failed to send notification email',
       );
+      counter.add(1, { channel: 'email' });
```

Comment:
Oh I see you export the counter :) 
That makes more sense now.

Reply:
Type safety rules!!